third-party/realdds/doc/DDS Use cases.md

```diff
@@ -0,0 +1,255 @@
+**Trigger** - The **user** triggers this use-case upon demand.
+
+**Successful Result** - The **server** stops publishing *image* messages of the requested profiles.
+
+**Main Scenario**
+1. The **user** sends a *stop streaming* message with the profiles requested to close listed.
+2. The **server** stops publishing *image* messages for the requested profiles.
+
+**Variations**
+None.
+
+**Exceptions**
+1. The device is not streaming data of the requested profiles
+    1. Terminate the use-case. //TODO - send failure *notification*? Currently not in the ICD
+
+**Constraints**
+None.
+
+**Notes**
+1. Currently, when using librealsense API to control the device (as is the case with the rs-dds-server tool), when stoping a stream all streams in the containing sensor will be stopped. 
```

Comment:
The "Currently..." is not so clear.
Did you mean to say something like:
"The realdds library behaves the same way as librealsense API when stopping one of the streams: it stops all other streams in the containing sensor.

Reply:
No, I will try to rephrase.
realdds does not put limitations on active streams requested to stop. If Infrared 1 and Infrared 2 are both streaming you can request to stop only one of them and keep the other streaming. LibRS, and the `rs-dds-server` tool that uses it, will stop both streams even if you only request to stop one, because they are from the same `sensor`. (realdds does not have a `sensor` concept, only streams)

Reply:
ok